### PR TITLE
Fixes for types, docs, tests

### PR DESCRIPTION
**Pull request by sjlex**
Minor adjustments.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,6 +191,20 @@ const App = () => {
 ReactDOM.render(<App />, node); // "Alex" will be rendered
 ```
 
+### Templates
```
Comment (yankouskia) [inline README.md:194]: Nice, I think this place is better to describe hooks limitation after its documentation 👍

File: index.d.ts [hunk 2]
```diff
@@ -13,15 +13,15 @@ interface LocalizationContextValue {
...
+  locale?: string;
   disableCache?: boolean;
   translations: Record<string, unknown>;
 }
 
 interface MessageComponentProps {
   descriptor: string;
-  values: Record<string, unknown>;
-  defaultMessage: string;
+  values?: Record<string, unknown>;
```
Comment (yankouskia) [inline index.d.ts:23]: Let's declare and use interfaces for values and translations. Basically the types are not `unknown`:
```js
values?: Record<string, string | number>;

// ....

interface Translations {
  [key: string]: string | number | Translations;
}
```

**Comment by serjleks** [comment]
Thanks for the feedback!

**Comment by yankouskia** [comment]
@serjleks could you please also bump minor version, so I could merge and deploy new version?
Thank you for the work!

**Comment by yankouskia** [comment]
@serjleks thanks for contribution! Latest version is published!

**Pull request merged**